README.rst

```diff
@@ -42,14 +42,32 @@ path to the SQLite database.
 
     $ xbrl_extract {path_to_filings} {path_to_database}
 
+This repo contains a small selection of FERC Form 1 filings from 2021 in the
+``examples`` directory. To test the tool on these filings, use the command
+
+.. code-block:: console
+
+    $ xbrl_extract examples/ferc1-2021-sample.zip {path_to_database}
+
 By default, the CLI will use the 2022 version of the FERC Form 1 Taxonomy to create
 the structure of the output database. To specify a different taxonomy use the
-``--taxonomy`` option.
+``--taxonomy`` option. This can be either a URL pointing to a taxonomy entry point,
+or a path to a local zipfile.
 
 .. code-block:: console
 
     $ xbrl_extract {path_to_filings} {path_to_database} --taxonomy {url_of_taxonomy}
 
+If using a zipfile, you must also specify the path to the taxonomy entry point within
```

Comment:
"If you're using a zipfile..." for what purpose? To supply the filings? Or to supply the taxonomy? Or both?

Can you use a zipfile for the filings, and a URL for the taxonomy? Or if you're using a zipfile for the filings, do you **have** to use a path to the taxonomy within the zipfile?

If reading filings from within the zipfile means you *must* use a taxonomy from within the zipfile, then would it make sense to just use `--taxonomy` to specify either the URL (if not using a zipfile) or the path within the archive (if using a zipfile)?

Then once there's a standardized location within the zipfile for the taxonomy, the UI for both the "files / directory + URL" and the "zipfile + archived taxonomy" would be the same -- with no `--taxonomy` option, they'd just use the default location (either URL or archive).

Reply:
If it's possible to refer directly to a taxonomy URL on the command line I think we should show an example of that, probably pointing to the same FERC 1 taxonomy URL that it would use by default in the example.

Reply:
I've changed the wording to hopefully make this more clear. If the `path_to_filings` option points to a zipfile, the tool will by default only look for filings there. Technically the filings and the taxonomy can be contained in the same zipfile (as is the case for the archives used by PUDL), but you would still have to indicate that the taxonomy is contained in this zipfile using the `--taxonomy` option, and point to the taxonomy entry point with the `--archive-path` option. This is because a taxonomy is made up of many different files, and the tool doesn't know which of these is the entry point.

As I've added more functionality to the tool (like the ability to parse taxonomies from a zipfile) the CLI has gotten pretty clunky because I was mostly focused on enabling PUDL's usage, but I'd like to rework/streamline it at some point. I think I could allow it to automatically discover the taxonomy entry point, but I just haven't dedicated the time yet.